doc/running.rst

```diff
@@ -42,7 +42,8 @@ In order to avoid the slowdowns and warnings, users can direct the packages to c
 cache files in directories that are private to each rank by using the ``XDG_CACHE_HOME``
 environment variable, such as in the following example::
 
-   $ srun -n 512 bash -c 'XDG_CACHE_HOME=/p/lscratchh/diener3/xdg-scratch$SLURM_PROCID python examples/wave-eager-mpi.py'
+   $ export XDG_CACHE_HOME="/tmp/$USER/xdg-scratch"
+   $ srun -n 512 python examples/wave-eager-mpi.py'
```

Comment:
Will this automatically propagate env vars?

Reply:
In my tests, yes.